.github/workflows/basic.yml

```diff
@@ -128,6 +62,16 @@ jobs:
     - name: Checkout
       uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
+    - name: Install oneAPI basekit
```

Comment:
you could move this step below the `Install apt packages` step and omit the `apt update` command here

Reply:
done